airflow/post_install.sh

```diff
@@ -18,6 +18,11 @@ python $AIRFLOW_HOME/utils/set_default_variables.py \
     pipeline_dataset="{{ var.value.PIPELINE_DATASET }}" \
     events_dataset="{{ var.value.EVENTS_DATASET }}" \
     source_dataset="{{ var.value.PIPELINE_DATASET }}" \
+    publish_to_postgres="False" \
+    flexible_operator="bash" \
+    postgres_connection_string="postgresql://postgres:51aulOyho1OPIbOy@127.0.0.1:5432/api" \
```

Comment:
@smpiano remove this from the REPO, you never want to publish sensitive information in a repo that is not the deploy one. 

Also, remove `postgres_instance`, `postgres_table`, we are not including the line so you are asked to enter them when you setup a pipe.